src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEMethodSymbol.vb

```diff
@@ -1136,12 +1156,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             If Not _packedFlags.IsUseSiteDiagnosticPopulated Then
                 Dim errorInfo As DiagnosticInfo = CalculateUseSiteErrorInfo()
                 EnsureTypeParametersAreLoaded(errorInfo)
+                CheckUnmanagedCallersOnly(errorInfo)
                 Return InitializeUseSiteErrorInfo(errorInfo)
             End If
 
             Return _uncommonFields?._lazyUseSiteErrorInfo
         End Function
 
+        Private Sub CheckUnmanagedCallersOnly(ByRef errorInfo As DiagnosticInfo)
+            If Not _packedFlags.IsUnmanagedCallersOnlyPopulated Then
+                Dim hasUnmanagedCallersOnly As Boolean =
+                    DirectCast(ContainingModule, PEModuleSymbol).Module.FindLastTargetAttribute(_handle, AttributeDescription.UnmanagedCallersOnlyAttribute).HasValue
+                _packedFlags.SetIsUnmanagedCallersOnly(hasUnmanagedCallersOnly)
+            End If
+
+            If _packedFlags.IsUnmanagedCallersOnly Then
```

Comment:
>If _packedFlags.IsUnmanagedCallersOnly Then [](start = 12, length = 43)

It looks like this is the only place where use this flag and it actually "behind" ```IsUseSiteDiagnosticPopulated``` flag. Therefore, I don't thing we need those two new bits in ```_packedFlags```.

Reply:
> It looks like this is the only place where use this flag and it actually "behind" IsUseSiteDiagnosticPopulated flag. Therefore, I don't think we need those two new bits in _packedFlags.

This doesn't look addressed. We do not need to add any bits in my opinion. So both of them (two) should be removed.

---
In reply to: [566921884](https://github.com/dotnet/roslyn/pull/50886#discussion_r566921884) [](ancestors = 566921884)